k2/addons/k2-serial/zserial/serial.py

```diff
@@ -118,7 +194,7 @@ def before_command(self, message, messagebus):
     @sequential_dispatcher([SERIAL_SEND_COMMAND], [SERIAL_ENDPOINT], entity_option_id=SUT)
     def send_serial_command(self, message):
         data = message.data
-        if isinstance(message.data, SendSerialCommandData):
+        if isinstance(data, SendSerialCommandData):
```

Comment:
This change is unrelated to this review and could be merged in any case.

Reply:
good

Reply:
Added to master in cac4b28e9d576144faa43e77646bab1a4090dcd7